src/compiler/worker/runWebpackCompiler.js

```diff
@@ -67,7 +67,6 @@ module.exports = function runWebpackCompiler({
     ],
   });
 
-  // Use memory fs
   compiler.outputFileSystem = fs;
```

Comment:
Am I missing something or it seems like we could remove the `outputFileSystem` entirely? https://webpack.js.org/api/node/#custom-file-systems

Or at least we could simplify https://github.com/callstack/haul/pull/512/files#diff-99e77f1c262ae763f3d7d593b2c0b417R44 by spreading fs and adding `mkdirp` and `join` if that's necessary

Reply:
I added a comment here for future reference.